content-services/latest/install/ansible.md

```diff
@@ -0,0 +1,489 @@
+
+There are two types of installation: 
+* **Local** where all the components will be installed on the control node machine:
+
+![](../../images/deployment-type-local.png)
+
+* **Remote** (also known as `ssh`) where components can be installed on one or more remote `hosts`. These hosts can be bare metal machines, Virtual machines or instances running on a public cloud:
+
+![](../../images/deployment-type-ssh.png)
+
+
+## Prerequisites
+
+* For Alfresco Content Services Enterprise, credentials to access the necessary artefacts from [Nexus](https://artifacts.alfresco.com). Customers can request these through [Hyland Community](https://community.hyland.com/)
+
+
+# Setting up Ansible
+
+As mentioned in the introduction a control node is required to run the playbook. You can use any computer that has a Python installation as a control node; laptops, shared desktops, and servers can all run Ansible.
+
+In the interest of keeping this guide simple we will use an AWS EC2 instance as the control node, the steps required are shown below:
+
+1. Launch an EC2 instance using the Centos 7 or 8 (x86_64) AMI from the Marketplace (instance size/type does not matter)
+
+    ![](../../images/centos-ami.png)
+
+2. Download the Ansible playbook [zip file](https://nexus.alfresco.com/nexus/service/local/repositories/releases/content/org/alfresco/alfresco-ansible-deployment/1.0/alfresco-ansible-deployment-1.0.zip)
+
+3. Transfer the ZIP file to the control node and SSH into the machine
+
+    ```bash
+    scp -i <yourpem-file> <local-path>/alfresco-ansible-deployment-<version>.zip centos@<control-node-ip>:/home/centos/
+    ssh -i <yourpem-file> centos@<control-node-ip>
+    ```
+
+4. Install the required dependencies for Ansible (replace the 7 with 8 in the URL if you're using CentOS 8)
+
+    ```bash
+    sudo yum install -y unzip https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    ```
+
+5. Install Ansible
+
+    ```bash
+    sudo yum install -y ansible
+    ```
+
+6. Extract the ZIP file
+
+    ```bash
+    unzip alfresco-ansible-deployment-<version>.zip
+    ```
+
+7. Create environment variables to hold your Nexus credentials as shown below (replacing the values appropriately):
+
+    ```bash
+    export NEXUS_USERNAME="<your-username>"
+    export NEXUS_PASSWORD="<your-password>"
+    ```
+
+Without any additional configuration applied, the playbook will install the default Alfresco Content Services components. Please review the [configuration](#configure-your-installation) section below, to adjust some of the configurable installation options.
+
+To install everything on the control node follow the steps in the [Locahost Installation](#Local-Installation) section or to install to one or more other machines follow the steps in the [Remote Installation](#Remote-Installation) section.
```

Comment:
page links #Local-Installation and #Remote-Installation does not work, needs to be #local-installation an #remote-installation